### ISSUE TITLE
False negative for imports not at top of file when they're preceded by an `elif` since version 2.6.0

### ISSUE THREAD

**Issue by lukaszgo1** [PyCQA/pycodestyle#1005]
With the following small silly reproducer:
```
import sys
if sys.argv[1] == "test":
    print("Provided argument: test")
elif sys.argv[1] == "testing":
    print("Provided argument: testing")
import os
import ctypes
import json

```
pycodestyle reports no warnings about imports not at top of file since dd6268c488ec6ab23188f31f7a1d345a8e114cbe whereas with this commit reverted output is:
```
d:\pycodestyle_issue_elif.py:6:1: E402 module level import not at top of file
d:\pycodestyle_issue_elif.py:7:1: E402 module level import not at top of file
d:\pycodestyle_issue_elif.py:8:1: E402 module level import not at top of file

```

**Comment by sigmavirus24** [comment]
In the future, please search **closed and** open issues before creating new ones that are duplicates.

**Comment by lukaszgo1** [comment]
I've looked through open issues before opening this one, and after your comment have also looked at the closed issues by searching for `E402` and `elif` yet I'm unable to find a duplicate. It is certainly possible that someone reported the same problem but with STR so different to mine that I haven't realized it is a duplicate but in that case I'd appreciate if someone could point me in the right direction.
